config/default/pyproject.toml.j2

```diff
@@ -0,0 +1,26 @@
+[tool.towncrier]
+filename = "CHANGES.rst"
+directory = "news/"
+title_format = "{version} ({project_date})"
+underlines = ["-", ""]
+
+[[tool.towncrier.type]]
+directory = "breaking"
+name = "Breaking changes:"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "feature"
+name = "New features:"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "bugfix"
+name = "Bug fixes:"
+showcontent = true
+
```

Comment:
It is easier for developers to have these available and either ignore them or delete them, than to not have them at all. It is sad that documentation is ignored so much in Plone. Please add it.

"Internal" is used for CI/CD and other management tasks, such as running linters. Such tasks fit under no other existing types.

Reply:
Let me explain then :) mostly and foremost because I want to get this merged. There's always a next PR to amend the previous one 😉 

Second thing, as I mentioned, the good thing of having this project here is that one can customize each repository, I'm totally fine with having these options on all repositories, though I find it a bit pointless, that if we have a documentation repository and only a selected few packages do keep documentation, all of the other packages have to have that section as well, cal me picky if you will 😅

Reply:
The result of that decision is that repos then create their own names for common usage. I have seen `directory = "documentation"`, `directory = "docs"`, or `directory = "doc"` . Thus it is not standardized. To make it worse, many operation systems treat the `.doc` and `.docs` file extensions as Office documents, and instead of opening in an editor will open in an Office app.

The same could be said for "internal".

Overall, should this repo guide developers toward common standards, or let them reinvent the wheel with non-standardized implementations?

Please reconsider.

Reply:
sorry, that's already more times than enough that I already said that I'm NOT against it, could you please reconsider yourself if it is worth keeping this PR open for longer and longer preventing further work on it?

I already waited to merge this PR because I was choosing `codespell` rather than `vale` which was thrown over the fence and no hint on what that would entail, just to find that is way too much for single small repositories like the vast majority that make up plone. Now this with towncrier sections?

Quoting yourself: `Volto and a few other repos...` That does not sound that [plone.batching](https://github.com/plone/plone.batching/pull/33), [plone.subrequest](https://github.com/plone/plone.subrequest/pull/26) and [plone/plone.supermodel](https://github.com/plone/plone.supermodel/pull/43) would benefit from them at all.

And the point is that, as soon as it is added, we just run this script yet again on them and voilà, they have it, it's not a now or never thing.

Plone 6 Documentation is not finished but it is published, it will in a way, never be finished, just like this repo, that still has plenty of zope references that need to be polished. And I'm waiting for this one to start cleaning that for example...

Sorry if it sounds too direct, but your constant requests to reconsideration, where I clearly say that I'm __deferring__ it, not refusing, feel a bit harsh in this side of the screen.

I, only/mostly, have time at nights to work on Plone stuff, and having to wait 2 weeks to get this initial, __intentionally__ small first PR as a test bed (it is only being applied to 3 small packages!) is a bit soul crushing.

I plan to work on adding `zpretty`, `z3c.dependencychecker`, isolation/randomized testing, `pyroma` and other release checks stuff, maybe in-lining stuff from mr.roboto, etc etc, but if we can not even merge the first PR because it is lacking one or two options, we will hardly achieve much.

I already promised to work on it during Alpine City Sprint, I had hoped that before it much of it was already done in this repo if not spread already on the repositories themselves, as to have a ready to check implementation to discuss with the crowd that attends there, but at this peace we will hardly have this merged 😕 

Fun fact: I've spent almost a month at work collecting all our documentation, improving it, making guidelines on how to expand it, spearheading it within the online team and the administration department (that they all both will start using it from now on), updating our issue templates to always mention not only developers documentation, but also user facing documentation, to now be told that I don't care about documentation 🤷🏾

Reply:
When a suggestion in a code review is resolved without comment, that means the suggestion was not accepted, a big "nope". It does not mean "not now, but please resubmit your request (and it consequently discourages the submitter from engaging further, or if the submitter persists, it makes needless double work)."

> be told that I don't care about documentation

...but I said...

> It is sad that documentation is ignored so much in Plone.

That means not by *you* specifically, but by Plone's community in general. There are a few hard core documentarians (you know who you are 🤗 ), and I am grateful for their contributions. Although the Documentation Team and contributors have gone far in the last year, we still have a looooong way to go toward being close to completion for Plone 6. Based on our effort during the last year, I estimate it will take us at least another two full years of volunteer time. See the [state of what still needs to happen in Plone 6 Documentation](https://github.com/orgs/plone/projects/12/views/4), or focus only on the most critical piece of documentation [Installation](https://github.com/orgs/plone/projects/16/views/1), to see what I am talking about.

If we don't even prompt developers to write documentation in our tools for best practices, we are saying *as a community* "docs are unimportant, don't bother with them, just create features, bugfixes, and breaking items, that's good enough, we'll deal with it later, it's someone else's problem, mañana mañana." We should do everything possible to encourage and remind developers to write documentation, not defer or shut down that effort.

> Fun fact: I've spent almost a month at work

Fun fact: the Plone Documentation Team and contributors have been doing all that, too, except it's been for over a year as unpaid volunteers. I appreciate all of the contributions that are shared with the community.

Reply:
Sorry, I did not take time last week anymore to have a proper look at this PR.

I agree with Steve in that I would like to have `.documentation` and `.internal` as well. Note that documentation is not just "proper", full-blown documentation on readthedocs or similar, but also simply a README that gets a fix. So this can be relevant for all Plone repositories.

I agree with Gil in that this can be improved later. 99 percent of Plone packages currently have the three categories that are in this PR. So starting with this list is fine for me.
I have created https://github.com/plone/meta/issues/2 to not forget about this.